pkg/provider-local/webhook/dnsconfig/mutator.go

```diff
@@ -63,6 +63,7 @@ func (m *mutator) Mutate(ctx context.Context, newObj, oldObj client.Object) erro
 	}
 
 	metav1.SetMetaDataLabel(podMeta, local.LabelNetworkPolicyToIstioIngressGateway, v1beta1constants.LabelNetworkPolicyAllowed)
+	metav1.SetMetaDataLabel(podMeta, local.LabelNetworkPolicyToVirtualGarden, v1beta1constants.LabelNetworkPolicyAllowed)
```

Comment:
This should only be added to machine pods (and BBX?), right? DWD doesn't need this, does it?

Reply:
It is added for machine pods only now. Blackbox exporter already has such a label.